packages/artillery/lib/cmds/run-fargate.js

```diff
@@ -69,24 +58,40 @@ Examples:
 RunCommand.flags = {
   ...CommonRunFlags,
   count: Flags.integer({
-    description: runTestDescriptions.count
+    description: 'Number of load generator workers to launch'
   }),
   cluster: Flags.string({
-    description: runTestDescriptions.cluster
+    description: 'Name of the Fargate/ECS cluster to run the test on'
   }),
   region: Flags.string({
     char: 'r',
-    description: runTestDescriptions.region
+    description: 'The AWS region to run in',
+    options: supportedRegions,
+    default: 'us-east-1'
   }),
   secret: Flags.string({
-    multiple: true
+    multiple: true,
+    hidden: true
```

Comment:
No need to hide this, the flag works as expected if a Parameter Store value under `/artilleryio` path exists with that name.